cypress/e2mock/edge-application/list-edge-applications.cy.js

```diff
@@ -0,0 +1,65 @@
+import selectors from '../../support/selectors'
+
+describe('Edge Application List Spec', { tags: ['@dev2'] }, () => {
+  beforeEach(() => {
+    cy.login()
+    cy.intercept('GET', '/api/v4/edge_application/*', {
+      fixture: '/edge-application/edge-applications.json'
+    }).as('edgeApplicationsListApi')
+
+    cy.openProduct('Edge Application')
+
+    cy.wait('@edgeApplicationsListApi', { timeout: 30000 })
+  })
+
+  it('Should filter the list of edge applications.', function () {
+    cy.intercept('GET', '/api/v4/edge_application/*', (req) => {
+      if (req.url.includes('search=foo+1')) {
+        req.reply({
+          fixture: '/edge-application/edge-applications-filtered.json'
+        })
+      }
+    }).as('filteredEdgeApplicationsListApi')
+
+    cy.get(selectors.list.searchInput).clear()
+    cy.get(selectors.list.searchInput).type('foo 1{enter}')
+
+    cy.wait('@filteredEdgeApplicationsListApi')
+      .its('response.body')
+      .should('have.property', 'count', 1)
+    cy.get(selectors.list.filteredRow.column('name')).should('have.text', 'foo 1')
+  })
+
+  it('should ordering edge applications list', function () {
+    cy.intercept('GET', '/api/v4/edge_application/*', (req) => {
+      if (req.url.includes('ordering=name')) {
+        req.reply({
+          fixture: '/edge-application/edge-applications-ordering-name.json'
+        })
+      }
+    }).as('filteredEdgeApplicationsListApi')
+
+    cy.get(selectors.list.orderingHeader.firstColumn).click()
+
+    cy.wait('@filteredEdgeApplicationsListApi').then((interception) => {
+      expect(interception.response.body.results[0].name).to.eq('foo 4')
```

Comment:
This sounds a bit weird ...
We are ordering by name (ASC) right? So shouldn't we be expecting `foo 1` over here?